lib/inputs/adzerk-impressions.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const url = require('url');
+const logger = require('../logger');
+const pingurl = require('../pingurl');
+
+/**
+ * Hit adzerk impression urls
+ */
+module.exports = class AdzerkImpressions {
+
+  constructor(records) {
+    this._records = (records || []).filter(r => this.check(r));
+  }
+
+  check(record) {
+    return !!record.impressionUrl && !record.isDuplicate;
+  }
+
+  insert() {
+    if (this._records.length == 0) {
+      return Promise.resolve([]);
+    } else {
+      let hostCounts = {};
```

Comment:
Yeah, I was thinking this could be a better way to "benchmark" how many pingbacks/impressions were going out.  Group them by hostname, and count/total/graph them via CloudWatch log metrics.

Haven't actually tried counting log-events with values > 1 though - so I may need to switch to a json-logging format to make that work.  TBD!